keras/legacy/interfaces.py

```diff
@@ -601,7 +601,8 @@ def generator_methods_args_preprocessor(args, kwargs):
 legacy_model_constructor_support = generate_legacy_interface(
     allowed_positional_args=None,
     conversions=[('input', 'inputs'),
-                 ('output', 'outputs')])
+                 ('output', 'outputs'),
+                 ('label', 'labels')])
```

Comment:
crashed without it, the legacy converter doesn't support params that didn't exist before.